### DISCUSSION TITLE
DDM use prequential error?

### DISCUSSION THREAD

**Discussion by albertfrancajosuacosta** [online-ml/river#1204] [Q&A]
Hey;

The river`s implementation of DDM use prequential error?

I looked at the source code and understood that the implementation uses the error average.

` # Probability of error/failure
        self._p = stats.Mean()
....

       p_i = self._p.get()

`

The self._p.get() is

`   def get(self):
        return self._mean`

Thank you!

**Comment by MaxHalford** [comment]
Hey there. Can you put more effort into formatting your post? Could you also provide a bit more context/explanation? Currently I don't understand enough to be able to help you.
